mesonbuild/modules/gnome.py

```diff
@@ -203,6 +203,8 @@ class MkEnums(_MkEnumsCommon):
         vtail: T.Optional[str]
         depends: T.List[T.Union[BuildTarget, CustomTarget, CustomTargetIndex]]
 
+    ToolType = T.Union['Executable', ExternalProgram, 'OverrideProgram']
```

Comment:
I would remove the quotes around types. Seems inconsistent in various places now.

Reply:
done